boards/arm/nrf9160_innblue11/Kconfig.defconfig

```diff
@@ -0,0 +1,65 @@
+# Kconfig - nRF91 innblue v1.1 board configuration
+#
+# Copyright (c) 2018 innblue
+
+if BOARD_NRF9160_INNBLUE11 || BOARD_NRF9160_INNBLUE11NS
+
+config BOARD
+	default "nrf9160_innblue11"
+
+if ADC
+
+config ADC_0
+	default y
+
+endif # ADC
+
+if I2C
+
+config I2C_2
+	default y
+
+endif # I2C
+
+if PWM
+
+config PWM_0
+	default y
+
+endif # PWM
+
+if SPI
+
+config SPI_3
+	default y
+
+endif # SPI
+
+# For the secure version of the board the firmware is linked at the beginning
+# of the flash, or into the code-partition defined in DT if it is intended to
+# be loaded by MCUboot. If the secure firmware is to be combined with a non-
+# secure image (TRUSTED_EXECUTION_SECURE=y), the secure FW image shall always
+# be restricted to the size of its code partition.
+# For the non-secure version of the board, the firmware
+# must be linked into the code-partition (non-secure) defined in DT, regardless.
+# Apply this configuration below by setting the Kconfig symbols used by
+# the linker according to the information extracted from DT partitions.
+
+if BOARD_NRF9160_INNBLUE11 && TRUSTED_EXECUTION_SECURE
+
+config FLASH_LOAD_SIZE
+	default $(dt_hex_val,DT_CODE_PARTITION_SIZE)

```

Comment:
`dt_hex_val` function is deprecated, shouldn't it be `default $(dt_chosen_reg_size,$(DT_CHOSEN_Z_CODE_PARTITION))`